src/app/submission/components/submit/submit.component.ts

```diff
@@ -106,6 +109,26 @@ export class SubmitComponent implements OnInit {
       );
   }
 
+  requestConvertToSCEA() {
+    this.ingestService.put(this.convertLinkSCEA, undefined)
+      .subscribe(
+        res => {
+          setTimeout(() => {
+              this.alertService.clear();
+              this.loaderService.display(false);
+              this.alertService.success('', 'Your SCEA submission should start exporting shortly.');
```

Comment:
It doesnt export to SCEA though right? It just creates the files neccesary to do so